lib/rules/web/admin_console/4.2/user_management.xyaml

```diff
@@ -88,3 +88,32 @@ check_cluster_role_reference:
     resource_name: <role_name>
     resource_link: /k8s/cluster/clusterroles/<role_name>
   action: check_resource_item
+check_no_create_button_on_list_page:
```

Comment:
Can we rename it to `check_no_create_button_on_pods_list_page` since we used `goto_projects_pods_list_page` , that would be much clearer when reading from feature